ros_ign_dev/ign_gazebo_dev/cmake/ignition_gazebo_dev-extras.cmake

```diff
@@ -0,0 +1,17 @@
+# This code here calls and sets required igniton gazebo based on ROS Distribution so,packages can directly go find_package(ign_gazebo_dev) so,that 
+# they don't need to change code with different ros distros
+if(DEFINED ENV{ROS_DISTRO})    
+message(STATUS "Detected ROS Distro is $ENV{ROS_DISTRO}")
+
+if("$ENV{ROS_DISTRO}" MATCHES "foxy")
+  find_package(ignition-gazebo3 REQUIRED)
+  set(IGN_GAZEBO_VER ${ignition-gazebo3_VERSION_MAJOR})
+elseif("$ENV{ROS_DISTRO}" MATCHES "galactic")
+  find_package(ignition-gazebo5 REQUIRED)
+  set(IGN_GAZEBO_VER ${ignition-gazebo5_VERSION_MAJOR})
+elseif("$ENV{ROS_DISTRO}" MATCHES "humble")
+  find_package(ignition-gazebo5 REQUIRED)
+  set(IGN_GAZEBO_VER ${ignition-gazebo6_VERSION_MAJOR})
+else()
```

Comment:
It is probably worth allowing an override / fallback for unofficial ROS distributions.

Using a CMake variable like `OVERRIDE_ROS_GZ_VERSION` or similar for override or checking `ROS_GZ_VERSION` environment variable to fall back to if no known ROS_DISTRO is found.

Reply:
I have added ign fortress as an override version in ignition_gazebo_dev-extras.cmake and package.xml.